apps/app/components/pages/single-page-block.tsx

```diff
@@ -441,7 +448,11 @@ export const SinglePageBlock: React.FC<Props> = ({ block, projectDetails, index,
                     />
                   </div>
                   {block?.description_stripped.length > 0 && (
-                    <p className="mt-3 h-5 truncate text-sm font-normal text-custom-text-200">
+                    <p
+                      className={`mt-3 text-sm font-normal text-custom-text-200 ${
+                        showBlockDetails ? "" : "h-5 truncate"
```

Comment:
Render the description without truncating it and properly formatted using Remirror.